pimd/pim6_cmd.c

```diff
@@ -42,7 +42,71 @@
 #include "pimd/pim6_cmd_clippy.c"
 #endif
 
+DEFPY (show_ipv6_pim_rp,
+       show_ipv6_pim_rp_cmd,
+       "show ipv6 pim [vrf NAME] rp-info [json]",
+       SHOW_STR
+       IPV6_STR
+       PIM_STR
+       VRF_CMD_HELP_STR
+       "PIM RP information\n"
+       JSON_STR)
+{
+	struct vrf *vrf_struct =
+		vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
+	bool uj = use_json(argc, argv);
+	json_object *json = NULL;
+
+	if (!vrf_struct)
+		return CMD_WARNING;
+
+	if (uj)
+		json = json_object_new_object();
+
+	pim_rp_show_information(vrf_struct->info, vty, json);
+
+	if (uj)
+		vty_json(vty, json);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (show_ipv6_pim_rp_vrf_all,
+       show_ipv6_pim_rp_vrf_all_cmd,
+       "show ipv6 pim vrf all rp-info [json]",
+       SHOW_STR
+       IPV6_STR
+       PIM_STR
+       VRF_CMD_HELP_STR
+       "PIM RP information\n"
+       JSON_STR)
+{
+	bool uj = use_json(argc, argv);
```

Comment:
Since you are using DEFPY here, there is no need to use this api use_json. You can directly check from the command input if json is null or not.